src/components/EventCalendar.vue

```diff
@@ -98,9 +99,10 @@ provide("CalendarConfig", config);
 const emit = defineEmits<{
   (e: "event-created", event: CalendarEvent): void;
   (e: "event-clicked", event: CalendarEvent): void;
+  (e: "event-updated", event: CalendarEvent): void;
+  (e: "event-contextmenu", event: CalendarEvent): void;
```

Comment:
```suggestion
  (e: "event-right-clicked", event: CalendarEvent): void;
```
Although most likely scenario is a context menu, I think this might be clearer.